app/CMakeLists.txt

```diff
@@ -220,6 +220,20 @@ if (SPACES)
     target_link_libraries(native-lib loader::openxr_loader)
 endif()
 
+
+# Add dependency on tinygltf library, which is used to load hand models
+# from .glb assets. Since it is only a single C++ source and header,
+# we include them as native-lib's own build sources.
+add_definitions(-DTINYGLTF_ANDROID_LOAD_FROM_ASSETS)
+target_sources(
+    native-lib
+    PUBLIC
```

Comment:
Nit let's make this a bit more compact, we can put everything but the path in the same line

Reply:
Ok